pysatNASA/instruments/ses14_gold.py

```diff
@@ -189,10 +189,18 @@ def load(fnames, tag=None, inst_id=None):
                         [dt.datetime.strptime(str(val), "b'%Y-%m-%dT%H:%M:%SZ'")
                          for val in data['scan_start_time'].values])
         data = data.swap_dims({'nscans': 'time'})
+        meta['time'] = meta['scan_start_time']
```

Comment:
The standard I have in the latest pysat is no metadata information for the main time. Would be fine for xarray but for pandas since time is only available via data.index there is a disconnect in names.

Reply:
Hmm.  leaving time without meta info produces the "Default Meta added" message.  Maybe I need to set the epoch?

Reply:
by latest I mean the `load_meta_proc` branch